js/lib/baseDomain.js

```diff
@@ -44,6 +47,11 @@ module.exports.getBaseDomain = function (hostname) {
     return baseDomain
   }
 
+  // If the hostname is a TLD, return '' for the base domain
+  if (hostname in publicSuffixes) {
+    return ''
```

Comment:
previously for an effective TLD like `co.uk`, this would return `co.uk` as the base domain. that seems incorrect so i just set it to the empty string.

note that some eTLDs like `github.io` are resolvable